src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -1,35 +1,101 @@
-        {score}
-      </Text>
-      <Text style={{fontSize: 12, fontWeight: '700', color: colors.white}}>Karma</Text>
-    </View>
+    <TouchableWithoutFeedback onPress={() => setIsTooltipShown(true)}>
+      <View
+        style={{
+          borderRadius: 32,
+          flexDirection: 'row',
+          justifyContent: 'center',
+          alignItems: 'center',
+          alignSelf: 'flex-start'
+        }}>
+        <View style={{marginRight: 8}}>
+          <CircleGradient size={normalize(28)} width={normalize(2.3)} fill={score}>
+            <FastImage
+              resizeMode={FastImage.resizeMode.center}
+              source={BetterScore}
+              style={{width: 28, height: 28}}
+            />
+          </CircleGradient>
+        </View>
+        <Text
+          style={{
+            fontWeight: '500',
+            fontSize: 24,
+            lineHeight: 36,
+            marginRight: 8
+          }}>{`${score} Karma`}</Text>
+        <Tooltip
+          allowChildInteraction={true}
+          isVisible={isTooltipShown}
+          placement={'bottom'}
+          backgroundColor={'rgba(0,0,0,0)'}
+          closeOnContentInteraction={false}
+          onClose={() => setIsTooltipShown(false)}
+          closeOnBackgroundInteraction={true}
+          contentStyle={{
+            borderRadius: 10,
+            padding: 16,
+            height: 'auto'
+          }}
+          content={
+            <View>
+              <Text
+                style={{
+                  fontWeight: '600',
+                  fontSize: normalize(16),
+                  lineHeight: normalize(24),
+                  marginBottom: normalize(8)
+                }}>
+                Your Karma Score
+              </Text>
+              <Text
+                style={{
+                  fontWeight: '400',
+                  fontSize: normalize(12),
+                  lineHeight: normalize(18),
+                  marginBottom: normalize(8),
+                  color: '#69707C'
+                }}>
+                The higher your Karma score, the higher your visibility on the platform. Get
+                rewarded for positive contributions to the community - and avoid being blocked by
+                others!
+              </Text>
+              <Text
+                style={{
+                  fontWeight: '600',
+                  paddingVertical: normalize(8),
+                  fontSize: normalize(14),
+                  lineHeight: normalize(20),
+                  color: '#4782D7',
+                  width: '100%',
+                  textAlign: 'right'
+                }}
+                onPress={() => setIsTooltipShown(false)}>
+                Got it!
+              </Text>
+            </View>
+          }>
+          <View style={{}}>
+            <MemoIcQuestionMark width={normalize(13)} height={normalize(13)} />
+          </View>
+        </Tooltip>
+      </View>
+    </TouchableWithoutFeedback>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The tooltip functionality added to the KarmaScore component is a nice touch for user experience. However, it seems like there are hardcoded styles within the component. It would be better to move these styles into a StyleSheet object outside of the component to improve readability and maintainability. This also allows for reusability of styles across different components.

```diff
+ const styles = StyleSheet.create({
+   container: {
+     borderRadius: 32,
+     flexDirection: 'row',
+     justifyContent: 'center',
+     alignItems: 'center',
+     alignSelf: 'flex-start'
+   },
+   circleGradientContainer: { marginRight: 8 },
+   karmaText: {
+     fontWeight: '500',
+     fontSize: 24,
+     lineHeight: 36,
+     marginRight: 8
+   },
+   // ... other styles
+ });

  export const KarmaScore = ({score}: KarmaScoreProps) => {
    const [isTooltipShown, setIsTooltipShown] = React.useState(false);
    return (
      <TouchableWithoutFeedback onPress={() => setIsTooltipShown(true)}>
        <View style={styles.container}>
          <View style={styles.circleGradientContainer}>
            <CircleGradient size={normalize(28)} width={normalize(2.3)} fill={score}>
              <FastImage
                resizeMode={FastImage.resizeMode.center}
                source={BetterScore}
                style={{width: 28, height: 28}}
              />
            </CircleGradient>
          </View>
          <Text style={styles.karmaText}>{`${score} Karma`}</Text>
          // ... other components
        </View>
      </TouchableWithoutFeedback>
    );
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->